src/Entity/DOI.php

```diff
@@ -38,6 +39,7 @@ class DOI extends Entity
      *
      * @Serializer\Groups({"doi"})
      */
+    #[SGroups(['export'])]
```

Comment:
I still would like this name changed. SGroups is confusing. Either use just Groups, and Maybe SerializerGroups